packages/api-clients/open-api/authorizationApi.yml

```
@@ -859,96 +864,686 @@ paths:
+  "/producerKeychains/{producerKeychainId}/users":
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    get:
+      tags:
+        - producerKeychain
+      summary: List Producer Keychain users
+      description: List Producer Keychain users
+      operationId: getProducerKeychainUsers
+      parameters:
+        - name: producerKeychainId
+          in: path
+          description: ID of Producer Keychain the users belong to
+          required: true
+          schema:
             type: string
             format: uuid
-      required:
-        - name
-        - members
-    Client:
-      description: Models Client details
-      type: object
-      properties:
-        id:
+      responses:
+        "200":
+          description: Request succeed
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Users"
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "404":
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  "/producerKeychains/{producerKeychainId}/users/{userId}":
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - name: producerKeychainId
+        in: path
+        description: The Producer Keychain id
+        required: true
+        schema:
           type: string
           format: uuid
-        name:
-          type: string
-        consumerId:
+      - name: userId
+        in: path
+        description: The identifier of the user between the security user and the consumer
+        required: true
+        schema:
           type: string
           format: uuid
-        createdAt:
-          type: string
-          format: date-time
-        purposes:
-          type: array
-          items:
-            type: string
-            format: uuid
-          default: []
-        description:
-          type: string
-        users:
-          type: array
-          items:
-            type: string
-            format: uuid
-          uniqueItems: true
-        kind:
-          $ref: "#/components/schemas/ClientKind"
-      required:
-        - id
-        - name
-        - consumerId
-        - purposes
-        - users
-        - kind
-        - createdAt
-    ClientKind:
-      type: string
-      description: kind of client
-      enum:
-        - CONSUMER
-        - API
-    ClientWithKeys:
+    delete:
+      description: Removes a user from a Producer Keychain
+      tags:
+        - producerKeychain
+      summary: Remove an user from a Producer Keychain
```

and other occurrences
```suggestion
      summary: Remove a user from a Producer Keychain
```